kolibri/plugins/learn/assets/src/views/ProgressBar.vue

```diff
@@ -9,18 +9,35 @@
     :aria-valuenow="progress * 100"
   >
     <p
-      v-if="completed"
```

Comment:
This will add an extraneous `<p>` tag in the case that it this is not a quiz and not completed. We should retain the `v-if` but change it to:
```
v-if="completed || isQuiz"
```

Reply:
I didn't notice this one